WalletWasabi.Tests/UnitTests/WalletDirTests.cs

```diff
@@ -157,21 +158,31 @@ public async Task GetNextWalletTestAsync()
 		{
 			var baseDir = Path.Combine(Global.Instance.DataDir, EnvironmentHelpers.GetCallerFileName(), EnvironmentHelpers.GetMethodName());
 			await CleanupWalletDirectoriesAsync(baseDir);
-
 			var walletDirectories = new WalletDirectories(baseDir);
-
-			Assert.Equal("Wallet0", walletDirectories.GetNextWalletName());
-
-			await File.Create(Path.Combine(walletDirectories.WalletsDir, $"Wallet0.json")).DisposeAsync();
-			await File.Create(Path.Combine(walletDirectories.WalletsDir, $"Wallet1.json")).DisposeAsync();
-			await File.Create(Path.Combine(walletDirectories.WalletsDir, $"Wallet3.json")).DisposeAsync();
-
-			// This should not matter.
-			await File.Create(Path.Combine(walletDirectories.WalletsBackupDir, $"Wallet2.json")).DisposeAsync();
-
-			Assert.Equal("Wallet2", walletDirectories.GetNextWalletName());
-
-			Assert.Equal("Foo0", walletDirectories.GetNextWalletName("Foo"));
+			IoHelpers.CreateEmptyFile(Path.Combine(walletDirectories.WalletsDir, "Random Wallet 3.json"));
```

Comment:
Files names with spaces are notorious source of various bugs in Bash ([like this](https://askubuntu.com/questions/343727/filenames-with-spaces-breaking-for-loop-find-command)) because it requires special handling to loop over such files. Not that people do that regularly with Wasabi JSON files.

But maybe it's worth considering whether those spaces are needed and whether replacing them with dots, underscores, camelCase or something is or is not a good idea. I'm not sure how mapping of wallet JSON files to names in GUI works. Maybe it would be user-unfriendly. It's just an observation. Feel free to ignore.

Reply:
On the other hand, we can use them boldly so the bugs will come out early. At the end, spaces cannot be avoided like the bug we had with windows user names and Tor data-dir. Better to prepare for spaces by default.